flake.nix

```diff
@@ -321,6 +321,12 @@
         common-gpu-nvidia-sync = import ./common/gpu/nvidia/prime-sync.nix;
         common-gpu-nvidia-nonprime = import ./common/gpu/nvidia;
         common-gpu-nvidia-disable = import ./common/gpu/nvidia/disable.nix;
+        common-gpu-nvidia-ada-lovelace = import ./common/gpu/nvidia/ada-lovelace;
```

Comment:
Same as in https://github.com/NixOS/nixos-hardware/pull/1198#discussion_r1808526854
We don't want to expose more common/ modules because of https://github.com/NixOS/nixos-hardware/issues/992
The linked PR also contains a work around on how to use them anyway, if you must.

Reply:
Alternative: Upstream them to nixpkgs and use them from there.